sqrl-server/sqrl-server-core/src/main/java/com/datasqrl/graphql/jdbc/JdbcExecutionContext.java

```diff
@@ -79,14 +79,13 @@ public CompletableFuture runQuery(ResolvedSqlQuery resolvedQuery, boolean isList
     if (preparedQueryContainer != null) {
       return CompletableFuture.supplyAsync(
           () -> {
-            try (PreparedStatement statement =
-                preparedQueryContainer
-                    .getConnection()
-                    .prepareStatement(preparedQueryContainer.getPreparedQuery())) {
+            try (final Connection conn = preparedQueryContainer.connection();
+                final PreparedStatement stmt =
+                    conn.prepareStatement(preparedQueryContainer.preparedQuery())) {
               for (int i = 0; i < paramObj.size(); i++) {
-                statement.setObject(i + 1, paramObj.get(i));
+                stmt.setObject(i + 1, paramObj.get(i));
               }
-              ResultSet resultSet = statement.executeQuery();
+              ResultSet resultSet = stmt.executeQuery();
```

Comment:
no need to change it now, but, I would prefer to see more `var`s as we refactor code